crates/holochain/src/conductor/tests/install_app_bundle.rs

```diff
@@ -424,6 +423,155 @@ async fn network_seed_regression() {
 #[cfg(feature = "glacial_tests")]
 #[ignore = "this is a really useful comprehensive test, but it's so dang slow"]
 async fn network_seed_affects_dna_hash_when_app_bundle_is_installed() {
+    use tempfile::TempDir;
+
+    struct TestcaseCommon {
+        conductor: SweetConductor,
+        dnas: Vec<DnaFile>,
+        tmp: TempDir,
+        _start: std::time::Instant,
+    }
+
+    #[derive(Clone, Copy, Debug, strum_macros::ToString)]
+    enum Location {
```

Comment:
I snuck this change in here, it's totally unrelated, just moving a bunch of feature-gated code to live under that feature gate to squelch warnings